Makefile

```diff
@@ -2,13 +2,24 @@
 #  OLM - Build and Test  #
 ##########################
 
+# setup-envtest on *nix uses XDG_DATA_HOME, falling back to HOME, as the default storage directory. Some CI setups
+# don't have XDG_DATA_HOME set; in those cases, we set it here so setup-envtest functions correctly. This shouldn't
+# affect developers.
+export XDG_DATA_HOME ?= /tmp/.local/share
```

Comment:
This does not work for some devs.

See:
- https://github.com/operator-framework/operator-controller/pull/803
- https://github.com/operator-framework/operator-controller/pull/804